app/views/two_factor_authentication/shared/max_login_attempts_reached.html.erb

```diff
@@ -0,0 +1,18 @@
+<% lockout_time_in_words = decorated_user.lockout_time_remaining_in_words %>
```

Comment:
I am not a huge fan of declaring vars in views like this (because I would expect this to be a helper method and woud look for it by grepping for `def lockout_time_in_words`), but this may be the style of the rest of the app